src/dashboard-widgets/dashboard-widgets.tsx

```diff
@@ -5,7 +5,8 @@ import Parcel from "single-spa-react/parcel";
 const patientDashboardParcels = [
   () => import("./basic-info/basic-info.parcel").then(m => m.default),
   () => import("./recent-visits/recent-visits.parcel").then(m => m.default),
-  () => import("../formentry/forms-parcel").then(m => m.default)
+  () => import("../formentry/forms-parcel").then(m => m.default),
+  () => System.import("@patient-weight-graph/seed")
```

Comment:
FYI - we have been naming our widgets with `@hackathon/patient-weight-graph`. The `@hackathon` part is the "namespace". See https://github.com/openmrs/openmrs-rfc-frontend/pull/11 for more details.

Could you please rename your project to something under the `@hackathon` namespace? You'll have to run `importMapOverrides.addOverride()` again in the browser console to getting it working.